docs/.vitepress/bootExamples.js

```diff
@@ -1,10 +1,14 @@
-import { buildWc } from './ex-base.js';
-import Button from '../buttons/Example.vue';
-import Alert from '../alert/Example.vue';
-import TextField from '../textfield/Example.vue';
-import TextArea from '../textarea/Example.vue';
+import { buildWc } from "./ex-base.js";
+import Alert from "../alert/Example.vue";
+import Button from "../buttons/Example.vue";
+import Expandable from "../expandable/Example.vue";
+import Input from "../input/Example.vue";
```

Comment:
This doesn't exist anymore so I guess the merge with the latest master will help here 🙌